gtk/src/image.rs

```diff
@@ -126,134 +125,134 @@ impl ImageBuilder {
     /// Build the [`Image`].
     #[must_use = "Building the object from the builder is usually expensive and is not expected to have side effects"]
     pub fn build(self) -> Image {
-        let mut properties: Vec<(&str, &dyn ToValue)> = vec![];
+        let mut builder = glib::Object::builder::<Image>();
```

Comment:
This should be a plain wrapper around the `ObjectBuilder` like the autogenerated builders now. Same for the other manual builders

Reply:
Oh nice!